ssh/proxy.go

```diff
@@ -452,6 +452,17 @@ func NewDownstreamConn(c net.Conn, config *ServerConfig) (*connection, error) {
 	fullConf := *config
 	fullConf.SetDefaults()
 
+	if len(fullConf.PublicKeyAuthAlgorithms) == 0 {
+		fullConf.PublicKeyAuthAlgorithms = supportedPubKeyAuthAlgos
```

Comment:
Set PublicKeyAuthAlgorithms by default(in common.go) after set default server config.